andela/activo-web
# Ft display edit building input field 159878982

**What does this PR do?**

This PR displays an input field where a user can edit a building name with save or cancel options.

**Description of Task to be completed?**

Given that  a user is on the options dropdown for a given building, when they  click on the edit option,
they should see an input field where they can edit a building name with save or cancel options.

**How should this be manually tested?**

After cloning the repo, navigate to `settings > centers > spaces` then click on the options dropdown for a given building. Then you should be able to see the edit option and upon clicking it, an input field should be displayed to edit the building. 

**Any background context you want to provide?**

xxx

**What are the relevant pivotal tracker stories?**

[#159878982 User should be able to see edit field for a building name](https://www.pivotaltracker.com/story/show/159878982)

**Screenshots (if appropriate)**

![image](https://user-images.githubusercontent.com/30326444/44779798-1f5f2b00-ab89-11e8-8a73-4e0bf39cab42.png)

**Questions:**

xxx
